node/tools/src/k8s.rs

```diff
@@ -21,6 +21,25 @@ pub async fn get_client() -> anyhow::Result<Client> {
     Ok(Client::try_default().await?)
 }
 
+/// Get a kube client
+pub async fn get_consensus_node_ips(client: &Client) -> anyhow::Result<Vec<String>> {
+    let pods: Api<Pod> = Api::namespaced(client.clone(), "consensus");
+    let lp = ListParams::default();
+    let pod = pods.list(&lp).await?;
+    let a: Vec<String> = pod
+        .into_iter()
+        .filter(|pod| {
+            pod.clone()
+                .metadata
+                .name
+                .unwrap()
+                .starts_with("consensus-node")
```

Comment:
Is this the most reliable way of selecting targets? Perhaps we can do an exact match on the docker image name?
Can we pass the prefix/image name as a command line argument?

Reply:
I liked the idea of matching the `consensus-node` Docker image, so I went with that approach.